deploy/helm/stac-scaler/templates/_checks.yaml

```
@@ -0,0 +1,7 @@
+################################
+    Container related checks
+################################
+
+{{- if .Values.repository }}
+ {{ required "The image repository must be specified" }}
+{{- end }}
```

Don't you need either `if not`, or just a single `required`?

```suggestion
{{ required "The image repository must be specified" .Values.repository }}
```